mapper/cumulocity/c8y_translator_lib/src/lib.rs

```diff
@@ -172,14 +179,14 @@ mod tests {
     #[test]
     fn check_thin_edge_translation_with_timestamp() {
         let single_value_thin_edge_json = r#"{
-                  "time" : "2013-06-22T17:03:14.000+02:00",
+                  "time" : "2013-06-22T17:03:14+02:00",
```

Comment:
The test expectations MUST not be changed during a refactoring.

The timestamp should be printed with a millisecond precision.

Reply:
Agree, but when I convert the RC3339 format timestamp string like here, DateTime::parse_from_rfc3339(&timestamp_str), strips of the milliseconds for some reason. I do not see any other API that can help here. Either we should parse string ourself of milliseconds and add later to fixedoffset time stamp.
Let me know if there any API that can help here.

Reply:
So update the test with non zero milli-seconds to make this obvious.